src/controllers/document.js

```diff
@@ -194,6 +194,7 @@ class DocumentController extends BaseController {
       action: 'search',
     };
 
+    options.verb = 'POST';
     for (const opt of ['from', 'size', 'scroll']) {
       request[opt] = options[opt];
       delete options[opt];
```

Comment:
I know it's not directly related to this PR but here the `options` object should not be mutated because the user may want to re-use it across subsequent calls